docs/content/getting-started/setup-openfga.mdx

```diff
@@ -27,43 +27,26 @@ This will start an HTTP server and gRPC server with the default configuration op
 
 ## Configuring The Server
 
-There are several configuration options for OpenFGA. One option is to pass them in with an environment file. For example:
-
-```
-$ cat ./myenv
-OPENFGA_DATASTORE_ENGINE=postgres
-OPENFGA_AUTH_PRESHARED_KEYS=key1,key2
-# etc
-
-$ docker run --env-file ./myenv openfga/openfga run
+You may configure the OpenFGA server with a `config.yaml` file. The `config.yaml` can be specified in either:
+- `/etc/openfga`
+- `$HOME/.openfga`
+- `.` (i.e., the current working directory).
+
+The OpenFGA will server search for the configuration file in the above order.
+
+Here is a sample configuration to run OpenFGA with postgres and using a preshared key for authentication:
+
+```yaml
+database:
```

Comment:
Should be `datastore` now. We just refactored this to be consistent within OpenFGA with our vernacular.

```
datastore:
    engine: postgres
    uri: postgres://user:password@localhost:5432/mydatabase
```